server/amp/handlers/visual-stories-bookend.js

```diff
@@ -0,0 +1,75 @@
+const get = require("lodash/get");
+
+function getImageUrl(story, config) {
+  if (get(story, ["story-template"]) === "news-elsewhere") {
+    return get(story, ["metadata", "reference-url"], "");
+  }
+  return `${config["sketches-host"]}/${story.slug}`;
+}
+
+async function bookendHandler(req, res, next, { config, client }) {
+  const relatedStoriesResponse = await client.getRelatedStories(
+    req.query.storyId,
+    req.query.sectionId
+  );
+  const relatedStories = relatedStoriesResponse["related-stories"];
```

Comment:
It could return early from here when `relatedStories.length == 0`

Reply:
yes, making change